Readme.md

```diff
@@ -0,0 +1,15 @@
+# SuperHotWinXRoyal777 🎰
+
+<img src="./images/Welcome.gif" alt="Welcome to SuperHotWinXRoyal777" width="100%"/>
```

Comment:
Not bad, you're on a right track. 

I'd take a more time to experiment around, maybe after this PR is merged.

Reply:
Feel free to feel free.

Reply:
Question: you put `<img ..` here instead of markdown image so you could set width to 100%?